### DISCUSSION TITLE
what is unpenalized partial log-likelihood?

### DISCUSSION THREAD

**Discussion by Foreist** [CamDavidsonPilon/lifelines#1365] [Q&A]
Are there any sources of partial likelihood metrics used in the framework?
It says "In this author's opinion, the best way to measure predictive performance is evaluating the log-likelihood on out-of-sample data."

**Comment by CamDavidsonPilon** [comment]
Hi @Foreist, can you expand on your question more? I'm not sure how to answer it.
